openstack/identity/v3/users/results.go

```diff
@@ -116,6 +116,13 @@ type AddToGroupResult struct {
 	gophercloud.ErrResult
 }
 
+// IsMemberOfGroupResult is the response from a IsMemberOfGroup operation. Call its
+// ExtractErr method to determine if the request succeeded or failed.
+type IsMemberOfGroupResult struct {
+	IsMember bool
+	gophercloud.ErrResult
```

Comment:
Let's use `gophercloud.Result` instead.

`ErrResult` is used exclusively with `ExtractErr()` and since that's no longer being done here, it would break the pattern.